package.json

```diff
@@ -37,7 +37,7 @@
     "method-override": "latest",
     "moment": "2.15.1",
     "mongodb": "^3.5.9",
-    "mongoose": "4.2.4",
+    "mongoose": "8.8.3",
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg" width="20"/></sub>&nbsp; Use of Hardcoded Credentials

Do not hardcode passwords in code. Found hardcoded password used in `password_token`. Test `the HTTP request body` flows into `redirect` 

<h6>
Line 31 | <a href="https://cwe.mitre.org/data/definitions/798.html">CWE-798</a> | <a href="https://cwe.mitre.org/data/definitions/259.html">CWE-259</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 574</a> | <a href="https://learn.dev.snyk.io/lesson/hidden-functionality/">Learn more about this vulnerability</h6>

<details>

<summary><b>Data flow: 3 steps</b></summary>

`Step 1` https://github.com/owner/repo/blob/2187ce28013c2862d62c3f7fea128055deeb459a/example.js#L47

`Step 2` [example.js#L51](https://github.com/owner/repo/blob/2187ce28013c2862d62c3f7fea128055deeb459a/example.js#L51)

`Step 3` https://github.com/owner/repo/blob/2187ce28013c2862d62c3f7fea128055deeb459a/example.js#L55

</details>